api/config/config_prod.json

```diff
@@ -30,7 +30,8 @@
     "projectId": "all-of-us-rw-prod"
   },
   "admin": {
-    "adminIdVerification": "workbench-id-verifications@researchallofus.org"
+    "adminIdVerification": "workbench-id-verifications@researchallofus.org",
+    "loginUrl": "http:\/\/all-of-us-workbench.appspot.com/login"
```

Comment:
Is this the right url?

Reply:
This is the right url: `https://all-of-us-rw-prod.appspot.com/login`